libs/core/models/user-secret.ts

```diff
@@ -0,0 +1,9 @@
+/** User secret. */
+export interface UserSecret {
+
+  /** Refresh token. */
+  readonly refresh: string;
+
+  /** Access token. */
+  readonly access: string;
```

Comment:
*refresh* and *access* are actually vebs, such naming for object properties could be confusing, don't you think?
```suggestion
  readonly refreshToken: string;

  /** Access token. */
  readonly accessToken: string;
```